Core/src/org/sleuthkit/autopsy/centralrepository/persona/PersonaDetailsPanel.java

```diff
@@ -712,6 +712,10 @@ void setMode(Component parent, PersonaDetailsMode mode, Persona persona) {
         }
         initializeFields();
     }
+    
+    public void setName(String name) {
```

Comment:
setName is already a method defined in the JPanel class

We should rename this to setPersonaName()